### PR TITLE
This is review for trainings

### PR DESCRIPTION
**Pull request by mohansgithub**
please check and comment

### CODE REVIEW

File: src/main/webapp/WEB-INF/views/home.jsp [hunk 1]
```diff
@@ -8,9 +8,10 @@
 	<div class="row-fluid">
 		<div class="span10 offset1">
 
-			<h1>
-Namma Anna Sharath
-			</h1>
+			<h1> Namma DevOps Class	</h1>
```
Comment (raghujavadi) [inline src/main/webapp/WEB-INF/views/home.jsp]: Update the code with date

**Review by raghujavadi — COMMENTED** [review]
Please check and update me

**Review by raghujavadi — APPROVED** [review]
Approve

**Review by vijindas — APPROVED** [review]
Looks good and approved

File: src/main/webapp/WEB-INF/views/home.jsp [hunk 2]
```diff
@@ -8,9 +8,10 @@
 	<div class="row-fluid">
 		<div class="span10 offset1">
 
-			<h1>
-Namma Anna Sharath
-			</h1>
+			<h1> Namma DevOps Class	11 02 2020</h1>
```
Comment (sharathgoud2016) [inline src/main/webapp/WEB-INF/views/home.jsp:11]: what purpose adding date

Comment (mohansgithub) [reply src/main/webapp/WEB-INF/views/home.jsp:11]: this is for tracking

**Comment by mohansgithub** [comment]
Two review passed merging code

**Pull request closed without merging**